python/desc/imsim/skyModel.py

```diff
@@ -87,46 +169,62 @@ def addNoiseAndBackground(self, image, bandpass=None, m5=None,
         @param [out] the input image with the background and noise model added to it.
         """
 
-        # calculate the sky background to be added to each pixel
+        #obs_db = 'minion_1016_sqlite_new_dithers.db'
+        #conn = sqlite3.connect(obs_db)
+        #c = conn.cursor()
+        #c.execute("SELECT obsHistID, moonPhase, dist2Moon, moonAlt FROM ObsHistory WHERE airmass>0")
+        #rows = c.fetchall()
+	
+	#obsid = 7177
+
+        #gen = ObservationMetaDataGenerator(database=obs_db, driver='sqlite')
+        #obs_md = gen.getObservationMetaData(obsHistID=rows[obsid][0], boundLength=3)[0]
+
+        camera = LsstSimMapper().camera
```

Comment:
There is now a method `lsst.sims.coordUtils.lsst_camera` that is designed such that, the first time you call `lsst_camera()` anywhere, it calls `LsstSimMapper().camera` and stores the result.  All subsequent calls to `lsst_camera()` return the cached result.  We should start using this function, as it will avoid the risk of us accidentally calling the expensive `LsstSimMapper().camera` more than once.